**Linux** [Build 9549](http://brontes.lab.eng.brq.redhat.com/viewLog.html?buildId=92840&buildTypeId=WFPR) outcome was **FAILURE** using a merge of 25d237016c76f95edc08c0cbbed664d6fed0c2c4
Summary: Tests failed: 1 (1 new), passed: 3305, ignored: 194 Build time: 00:58:07
### Failed tests
```
org.jboss.as.test.integration.ee.datasourcedefinition.DataSourceDefinitionTestCase: 	at org.jboss.as.arquillian.container.ArchiveDeployer.deployInternal(ArchiveDeployer.java:83)
	at org.jboss.as.arquillian.container.ArchiveDeployer.deployInternal(ArchiveDeployer.java:64)
	at org.jboss.as.arquillian.container.ArchiveDeployer.deploy(ArchiveDeployer.java:46)
	at org.jboss.as.arquillian.container.CommonDeployableContainer.deploy(CommonDeployableContainer.java:144)
Caused by: java.lang.Exception: {"WFLYCTL0080: Failed services" => {"jboss.data-source.\"jboss.naming.context.java.module.testds.testds.dse\"" => "org.jboss.msc.service.StartException in service jboss.data-source.\"jboss.naming.context.java.module.testds.testds.dse\": WFLYJCA0033: Error during the deployment of jboss.naming.context.java.module.testds.testds.dse
    Caused by: org.jboss.msc.service.StartException in anonymous service: WFLYJCA0032: Unable to start the ds because it generated more than one cf"}}
	at org.jboss.as.controller.client.helpers.standalone.impl.ServerDeploymentPlanResultFuture.getActionResult(ServerDeploymentPlanResultFuture.java:134)
	at org.jboss.as.controller.client.helpers.standalone.impl.ServerDeploymentPlanResultFuture.getResultFromNode(ServerDeploymentPlanResultFuture.java:123)
	at org.jboss.as.controller.client.helpers.standalone.impl.ServerDeploymentPlanResultFuture.get(ServerDeploymentPlanResultFuture.java:85)
	at org.jboss.as.controller.client.helpers.standalone.impl.ServerDeploymentPlanResultFuture.get(ServerDeploymentPlanResultFuture.java:42)
	at org.jboss.as.controller.client.helpers.standalone.ServerDeploymentHelper.deploy(ServerDeploymentHelper.java:55)
	at org.jboss.as.arquillian.container.ArchiveDeployer.deployInternal(ArchiveDeployer.java:77)
	at org.jboss.as.arquillian.container.ArchiveDeployer.deployInternal(ArchiveDeployer.java:64)
	at org.jboss.as.arquillian.container.ArchiveDeployer.deploy(ArchiveDeployer.java:46)
	at org.jboss.as.arquillian.container.CommonDeployableContainer.deploy(CommonDeployableContainer.java:144)
------- Stdout: -------
 [0m14:32:41,648 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-4) WFLYSRV0028: Stopped deployment EEConcurrentManagementTestCase.jar (runtime-name: EEConcurrentManagementTestCase.jar) in 2ms
 [0m [0m14:32:41,652 INFO  [org.jboss.as.repository] (management-handler-thread - 4) WFLYDR0002: Content removed from location /opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/basic/target/jbossas/standalone/data/content/a7/dab4eb314616da16da9850693badaa861e51eb/content
 [0m [0m14:32:41,652 INFO  [org.jboss.as.server] (management-handler-thread - 4) WFLYSRV0009: Undeployed "EEConcurrentManagementTestCase.jar" (runtime-name: "EEConcurrentManagementTestCase.jar")
 [0m [0m14:32:41,669 INFO  [org.jboss.as.repository] (management-handler-thread - 2) WFLYDR0001: Content added at location /opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/basic/target/jbossas/standalone/data/content/b4/4598e808c0dc107595a8b2f325b8b4a0ca15b7/content
 [0m [0m14:32:41,670 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-4) WFLYSRV0027: Starting deployment of "testds.war" (runtime-name: "testds.war")
 [0m [33m14:32:41,680 WARN  [org.jboss.as.dependency.private] (MSC service thread 1-4) WFLYSRV0018: Deployment "deployment.testds.war" is using a private module ("org.wildfly.security.manager:main") which may be changed or removed in future versions without notice.
 [0m [0m14:32:41,695 INFO  [org.jboss.weld.deployer] (MSC service thread 1-4) WFLYWELD0003: Processing weld deployment testds.war
 [0m [0m14:32:41,697 INFO  [org.jboss.as.ejb3.deployment] (MSC service thread 1-4) WFLYEJB0473: JNDI bindings for session bean named 'DataSourceBean' in deployment unit 'deployment "testds.war"' are as follows:

	java:global/testds/DataSourceBean!org.jboss.as.test.integration.ee.datasourcedefinition.DataSourceBean
	java:app/testds/DataSourceBean!org.jboss.as.test.integration.ee.datasourcedefinition.DataSourceBean
	java:module/DataSourceBean!org.jboss.as.test.integration.ee.datasourcedefinition.DataSourceBean
	java:global/testds/DataSourceBean
	java:app/testds/DataSourceBean
	java:module/DataSourceBean

 [0m [0m14:32:41,749 INFO  [org.jboss.weld.deployer] (MSC service thread 1-4) WFLYWELD0006: Starting Services for CDI deployment: testds.war
 [0m [0m14:32:41,750 INFO  [org.jboss.as.arquillian] (MSC service thread 1-4) Arquillian deployment detected: ArquillianConfig[service=jboss.arquillian.config."testds.war",unit=testds.war,tests=[org.jboss.as.test.integration.ee.datasourcedefinition.DataSourceDefinitionTestCase]]
 [0m [31m14:32:41,752 ERROR [org.jboss.as.connector.subsystems.datasources.AbstractDataSourceService$AS7DataSourceDeployer] (MSC service thread 1-4) Error during the deployment of java:comp/dse: javax.resource.ResourceException: IJ031103: At least one connection property must be defined for datasource-class: org.jboss.as.test.integration.ee.datasourcedefinition.EmbeddedDataSource
	at org.jboss.jca.adapters.jdbc.local.LocalManagedConnectionFactory.createConnectionFactory(LocalManagedConnectionFactory.java:108)
	at org.jboss.jca.deployers.common.AbstractDsDeployer.deployDataSource(AbstractDsDeployer.java:772)
	at org.jboss.jca.deployers.common.AbstractDsDeployer.createObjectsAndInjectValue(AbstractDsDeployer.java:312)
	at org.jboss.as.connector.subsystems.datasources.AbstractDataSourceService$AS7DataSourceDeployer.deploy(AbstractDataSourceService.java:364)
	at org.jboss.as.connector.subsystems.datasources.AbstractDataSourceService.start(AbstractDataSourceService.java:145)
	at org.jboss.msc.service.ServiceControllerImpl$StartTask.startService(ServiceControllerImpl.java:1948)
	at org.jboss.msc.service.ServiceControllerImpl$StartTask.run(ServiceControllerImpl.java:1881)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)

 [0m [31m14:32:41,753 ERROR [org.jboss.msc.service.fail] (MSC service thread 1-4) MSC000001: Failed to start service jboss.data-source."jboss.naming.context.java.module.testds.testds.dse": org.jboss.msc.service.StartException in service jboss.data-source."jboss.naming.context.java.module.testds.testds.dse": WFLYJCA0033: Error during the deployment of jboss.naming.context.java.module.testds.testds.dse
	at org.jboss.as.connector.subsystems.datasources.AbstractDataSourceService.start(AbstractDataSourceService.java:159)
	at org.jboss.msc.service.ServiceControllerImpl$StartTask.startService(ServiceControllerImpl.java:1948)
	at org.jboss.msc.service.ServiceControllerImpl$StartTask.run(ServiceControllerImpl.java:1881)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)


```

--- a/pom.xml
+++ b/pom.xml
@@ -160,7 +160,7 @@
         <version.org.jboss.ejb3.ext-api>2.2.0.Final</version.org.jboss.ejb3.ext-api>
         <version.org.jboss.genericjms>1.0.7.Final</version.org.jboss.genericjms>
         <version.org.jboss.iiop-client>1.0.0.Final</version.org.jboss.iiop-client>
-        <version.org.jboss.ironjacamar>1.3.2.Final</version.org.jboss.ironjacamar>
+        <version.org.jboss.ironjacamar>1.3.3.Final</version.org.jboss.ironjacamar>
         <version.org.jboss.jandex>2.0.1.Final</version.org.jboss.jandex>
         <version.org.jboss.jboss-transaction-spi>7.3.0.Final</version.org.jboss.jboss-transaction-spi>
         <version.org.jboss.metadata>10.0.0.Final</version.org.jboss.metadata>

--- a/testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ee/datasourcedefinition/DataSourceBean.java
+++ b/testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ee/datasourcedefinition/DataSourceBean.java
@@ -42,7 +42,8 @@ import java.sql.SQLException;
         ),
         @DataSourceDefinition(
                 name = "java:comp/dse",
-                className = "org.jboss.as.test.integration.ee.datasourcedefinition.EmbeddedDataSource"
+                className = "org.jboss.as.test.integration.ee.datasourcedefinition.EmbeddedDataSource",
+                url = "jdbc:embedded:/some/url"
         )
 }
 )

--- a/testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ee/datasourcedefinition/EmbeddedDataSource.java
+++ b/testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ee/datasourcedefinition/EmbeddedDataSource.java
@@ -74,6 +74,9 @@ public class EmbeddedDataSource implements DataSource {
         return null;
     }
 
+    public void setUrl(String v) {
+    }
+
     @Override
     public boolean isWrapperFor(Class<?> iface) throws SQLException {
         return false;

